src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -743,3 +743,16 @@ def _copy(self, **kwargs):
         }
         logger.debug(f"Run init params: {init_params}")
         return Run(**init_params)
+
+    @functools.cached_property
+    def _flow_type(self) -> str:
+        """Get flow type of run."""
+
+        from promptflow import load_flow
+        from promptflow._constants import FlowType
+        from promptflow._sdk.entities._eager_flow import EagerFlow
+
+        flow_obj = load_flow(source=self.flow)
```

Comment:
Catch the possible exception and return unknown?

Reply:
Actually, I don't want to catch here, but I should catch upstream where I need it, and I have already caught upstream.

Because I don't want to hide the errors thrown by **load_flow**, and also, if we catch the error and return **unknown**, it will cause **@functionalits.cached_property** to cache **unknown**, but I don't want the erroneous results to be cached.